.github/workflows/KubernetesInternalClients.yaml

```diff
@@ -46,7 +46,8 @@ jobs:
       - name: Setup .NET Core 3.1.x
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 3.1.x
+          dotnet-version: |
```

Comment:
Do we need to change KubernetesExternalClients yaml workflow?

Reply:
You are right, pushed the changes.